src/controllers/issuer.ts

```diff
@@ -26,11 +25,11 @@ export class IssuerController {
 			.isString()
 			.isIn([VerificationMethods.Ed255192020, VerificationMethods.Ed255192018, VerificationMethods.JWK])
 			.withMessage('Invalid verificationMethod'),
-		check('methodSpecificIdAlgo')
+		check('identifierFormatType')
 			.optional()
 			.isString()
 			.isIn([MethodSpecificIdAlgo.Base58, MethodSpecificIdAlgo.Uuid])
-			.withMessage('Invalid methodSpecificIdAlgo'),
+			.withMessage('Invalid identifierFormatType'),
 		check('network')
 			.optional()
 			.isString()
```

Comment:
Validation logic is incomplete, in terms of the union type suggested in next comments.

I'll leave it open, for future enhancement, unless @abdulla-ashurov finds it an easy / quick addition to the PR.

I'd suggest following credential status validation patterns.